tvx/suite_messages_paych.go

```diff
@@ -0,0 +1,228 @@
+package main
+
+import (
+	"encoding/json"
+	"os"
+
+	abi_spec "github.com/filecoin-project/specs-actors/actors/abi"
+	big_spec "github.com/filecoin-project/specs-actors/actors/abi/big"
+	paych_spec "github.com/filecoin-project/specs-actors/actors/builtin/paych"
+	crypto_spec "github.com/filecoin-project/specs-actors/actors/crypto"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/filecoin-project/oni/tvx/chain"
+	"github.com/filecoin-project/oni/tvx/drivers"
+)
+
+func MessageTest_Paych() error {
+	var initialBal = abi_spec.NewTokenAmount(200_000_000_000)
+	var toSend = abi_spec.NewTokenAmount(10_000)
+
+	err := func(testname string) error {
```

Comment:
can these tests be factored out to top level functions?

Reply:
Yes, I imagine this will be part of the refactor @raulk mentioned.